nemo/core/classes/modelPT.py

```diff
@@ -242,6 +243,8 @@ def restore_from(
                     # If override is top level config, extract just `model` from it
                     if 'model' in conf:
                         conf = conf.model
+                if override_data_dir is not None:
```

Comment:
@vladgets isn't data_dir part of config anyways?

Reply:
In Default config it is '???'. User should define it to refer to his dataset to train the model. We possibly had a strange issue with the model that I shared to Nvidia cloud is that I trained using my local data directory and it was automatically saved as a part of config in a .nemo file. Then when you download this model it refer to my local directory. So to use the pretrained model a user need redefine this parameter to his own data directory.

Reply:
Sorry about confusion, it's reverted now